tbxforms/static/sass/govuk-overrides/_settings.scss

```diff
@@ -0,0 +1 @@
+$tbxforms-font-family: arial, sans-serif !default; // Overridden to remove "GDS Transport" because tbxforms doesn't ship with that.
```

Comment:
I can't see we'd ever want a different font in our forms than what we set as the default for the project. Is there a way to not set a font at all and let it inherit?

Reply:
Good point. Would `unset` be a better option?

https://developer.mozilla.org/en-US/docs/Web/CSS/unset

Reply:
yep, sounds good

Reply:
Done in https://github.com/torchbox/tbxforms/pull/80/commits/d28b291a108132cc9621b1bafaf789a5675f3361